src/main/kotlin/model/PrisonerContentHub.kt

```diff
@@ -13,11 +13,12 @@ class PrisonerContentHub private constructor() {
     lateinit var model: Model
     lateinit var system: SoftwareSystem
     lateinit var contentHubFrontend: Container
+    lateinit var frontendProxy: Container
 
     /**
-     * TODO: add Prisoner internet infrastructure, AD
-     * TODO: add BT PINS
-     **/
+      * TODO: add Prisoner internet infrastructure, AD
+      * TODO: add BT PINS
+      **/
```

Comment:
It seems our formatters can't agree on where the `*` is 😂 
Shall we remove the TODOs in favour of GitHub issues/projects?